micrometer-core/src/test/java/io/micrometer/core/instrument/binder/jetty/JettyConnectionMetricsTest.java

```diff
@@ -44,12 +44,14 @@ class JettyConnectionMetricsTest {
 
     private Server server = new Server(0);
 
-    private ServerConnector connector = new ServerConnector(server);
+    private NetworkTrafficServerConnector connector = new NetworkTrafficServerConnector(server);
 
     private CloseableHttpClient client = HttpClients.createDefault();
 
     void setup() throws Exception {
-        connector.addBean(new JettyConnectionMetrics(registry));
+        JettyConnectionMetrics metrics = new JettyConnectionMetrics(registry);
+        connector.addBean(metrics);
+        connector.addNetworkTrafficListener(metrics);
```

Comment:
This looks like new setup that isn't documented in the class JavaDoc or done in the static `addToAllConnectors` method. It will be a behavior change that the bytes in/out will only be available if this is set, right?

Reply:
Yes, that's why I'm showing this in the tests that bytes are not equal to what they were used to.